application/controllers/Lotw.php

```diff
@@ -696,7 +696,7 @@ function lotw_download($sync_user_id = null) {
 				curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
 				curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 30);
 				$content = curl_exec($ch);
-				if(curl_errno($ch)){
+				if(curl_errno($ch) || str_contains($content,"Username/password incorrect</I>")) {
```

Comment:
This leads to a misleading display in case of wrong credentials (because of the added result code of curl which is "no error"):

![Screenshot from 2024-12-04 09-30-16](https://github.com/user-attachments/assets/c09f3a8b-2f77-4a7d-8bb1-f1927eb858ef)